ansible/roles/schulcloud-server-core/tasks/main.yml

```
@@ -142,6 +142,12 @@
       namespace: "{{ NAMESPACE }}"
       template: api-delete-s3-files-cronjob.yml.j2
 
+  - name: Delete Tldraw Files CronJob
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: tldraw-delete-files-cronjob.yml.j2
```

Missing condition that the cronjob only add if tldraw is activate
```suggestion
    kubernetes.core.k8s:
      kubeconfig: ~/.kube/config
      namespace: "{{ NAMESPACE }}"
      template: tldraw-delete-files-cronjob.yml.j2
    when: WITH_TLDRAW is defined and WITH_TLDRAW|bool
```